tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -17,6 +17,9 @@ readonly script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
 source "${script_dir}/../../scripts/lib.sh"
 
+readonly packaging_root_dir="${repo_root_dir}/tools/packaging"
+patches_path=""
```

Comment:
Ooi, where do we use this?

Reply:
We are calling get_config_version to get the kernel config version. And without these variables, we get unbound variable error.

Reply:
Hmm.  This seems the wrong place to fix this.

I will take some time to dig into this a little bit more and propose a similar solution, but in the part that I think is the correct one to be fixed.